lib/matplotlib/dates.py

```diff
@@ -222,6 +222,10 @@ def _to_ordinalf(dt):
         dt = dt.astimezone(UTC)
         tzi = UTC
 
+    if isinstance(dt, datetime.timedelta):
```

Comment:
If this goes through, you need to update the docstring for `_to_ordinalf` to include `timedelta`.